AssettoServer/Server/CMContentProviders/DefaultCMContentProvider.cs

```diff
@@ -17,4 +24,76 @@ public DefaultCMContentProvider(ACServerConfiguration configuration)
     {
         return ValueTask.FromResult(_configuration.ContentConfiguration);
     }
+
+    public void Initialize()
+    {
+        if (_configuration.ContentConfiguration == null) return;
+
+        if (_configuration.ContentConfiguration.Track != null)
+        {
+            var trackEntry = _configuration.Server.Track.Split('/').Last();
+            PrepareTrackDownload(trackEntry, _configuration.ContentConfiguration.Track);
+        }
+
+        if (_configuration.ContentConfiguration.Cars != null)
+        {
+            foreach (var car in _configuration.ContentConfiguration.Cars)
+            {
+                PrepareCarDownload(car.Key);
+            }
+        }
+    }
+
+    private void PrepareCarDownload(string carId)
+    {
+        if (_configuration.ContentConfiguration!.Cars!.TryGetValue(carId, out var car))
```

Comment:
I think it's better to pass the CMContentEntryCar as a parameter to this method. That way you don't have to use `!` here since you already checked for null in the calling method